hera_qm/xrfi.py

```diff
@@ -1490,6 +1491,10 @@ def xrfi_run_step(uv_files=None, uv=None, uvf_apriori=None,
         if True, calculate modified_z_score when computing overall z-score
         (only used if alg='overall_z_score').
         Default is False.
+    a_priori_flag_yaml : str, optional
```

Comment:
I'm having a hard time telling, but do the flags that are in the YAML get propagated to `uvf_apriori`? If not, they definitely should since that gets written to disk (which will provide a really nice check that things are working as expected).

Reply:
The flags do get propagated to `uvf_apriori` this is something the unittests check. They do not necessarily get propagated to metric derived flags since the metrics are computed in flagged channels?

Reply:
If these flags are applied to the UVCal/UVData object before calculating the metric, then they should get propagated to the other flag objects. Not 100% sure about that, but I don't think we need to worry about that now. As long as it appears in uvf_apriori, I'm happy.

Reply:
Sounds good.